### DISCUSSION TITLE
integration with flutter

### DISCUSSION THREAD

**Discussion by atiqfns** [tradingview/lightweight-charts#1324] [Q&A]
How to integrate tradeView with flutter?

**Comment by MannyIOI** [comment]
> How to integrate tradeView with flutter?

Did you find a way?

**Reply by mohammadreza1840** [reply]
Yes
Take look at https://github.com/louisnw01/lightweight-charts-python
in this repo used from js and python web view!!

**Comment by vishnuc** [comment]
You can use webview to load charts,, expo react native has good api now ,, where you can exchange data across your webview and nativeview aswel.
